utils/browser/Cargo.toml

```diff
@@ -16,10 +16,10 @@ js-sys = "0.3.34"
 wasm-bindgen = "0.2.57"
 wasm-bindgen-futures = "0.4.7"
 kvdb-web = "0.3"
-sc-cli = { path = "../../client/cli", default-features = false }
-sc-service = { path = "../../client/service", default-features = false }
-sc-network = { path = "../../client/network" }
-sc-chain-spec = { path = "../../client/chain-spec" }
+cli = { package = "sc-cli", path = "../../client/cli", default-features = false }
```

Comment:
Why renames suddenly?

Reply:
The other dependencies in `browser-utils` were renamed from `sc-x` to `x` in a merge, so I thought I'd do the same.

Reply:
There are now only 7 occurrences within Substrate where we rename the packages, 3 of which are here. So I think I'd not rename them.